Assets/HoloToolkit/Input/Scripts/CursorManager.cs

```diff
@@ -1,65 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I want us to be honest.  Keeping deprecated classes can cause confusion and creates a fragmented code base.  We have to move others forward with solid examples that can be built off of rather than having sentimental attachment to prior iterations.  If anyone wants to get older versions they are free to do so.